Sources/Features/Error/BacktraceError.swift

```diff
@@ -62,8 +66,18 @@ extension NetworkError {
 extension HttpError {
     var localizedDescription: String {
         switch self {
-        case .malformedUrl(let url): return "Provided URL cannot be parsed: \(url)."
-        case .unknownError: return "Unknown error occurred."
+        case .malformedUrl(let url):
+            return "Provided URL cannot be parsed: \(url)."
```

Comment:
small nit: can we either get rid of trailing dots or put the URL in quotes?
this goes for every description that has some data displayed in the string